### ISSUE TITLE
Compiling won't finish

### ISSUE THREAD

**Issue by Kompromittiert93** [thecodingmachine/workadventure#2065]
Hello,

looks like I'm hanging in the compiling process. I've been waited for more than 10 minutes. Also no CPU load is visible.

The last stdout, that I can see:

Starting compilation in watch mode...
maps_1    |
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/ButtonClickedEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/ChatEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/ClosePopupEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/EmbeddedWebsiteEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/EnterLeaveEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/GoToPageEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/HasPlayerMovedEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/IframeEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/LayerEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/LoadPageEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/LoadSoundEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/LoadTilesetEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/OpenCoWebSiteEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/OpenPopupEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/OpenTabEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/PlaySoundEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/SetTilesEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/SetVariableEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/UserInputChatEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/setPropertyEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/ui/MenuItemClickedEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/ui/MenuRegisterEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/Events/ui/TriggerActionMessageEvent.d.ts(1,21): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/iframe/IframeApiContribution.d.ts(1,26): error TS2307: Cannot find module 'generic-type-guard' or its corresponding type declarations.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/iframe/Sound/Sound.d.ts(1,22): error TS2503: Cannot find namespace 'Phaser'.
maps_1    | node_modules/@workadventure/iframe-api-typings/Api/iframe/room.d.ts(2,32): error TS2307: Cannot find module '../../Phaser/Map/ITiledMap' or its corresponding type declarations.
maps_1    |
maps_1    | 7:42:24 AM - Found 26 errors. Watching for file changes.

Host OS: AlmaLinux
SELinux disabled

**Comment by gparant** [comment]
Hi @Kompromittiert93, have you start `messages` service and can you check the logs of this service?
The service `messages` build all events broadcasted between front, pusher and back.

**Comment by moufmouf** [comment]
Wait... compilation is starting in "watch" mode. So it indeed never stops ;)

"docker-compose up" is not meant to compile and finish, but to compile and start the servers.

**Comment by Kompromittiert93** [comment]
I copied the contrib/docker/.env.template file again and the error was fixed. But now I have a new one.

I got the message that the connection cannot be established and was asked if I am connected to the Internet.

In the Dev-Tools (Google Chrome) it looks like, the connection to the maps service won't work. If I redirect directly to the URL from the maps service, I got a forbidden, so the webserver is listening and reachable.

@gparant I enabled the meassages-service, see attached screenshot

![pCtMn0JZsm](https://user-images.githubusercontent.com/99262238/162915887-1987fe42-dfe3-41a2-8011-7ffc610cce13.jpg)
